### DISCUSSION TITLE
Email Templates API

### DISCUSSION THREAD

**Discussion by CanRau** [supabase/supabase#9266] [Feature Requests]
Hey, I would love to be able to customize auth email templates via API, whether SDK, SQL, API doesn't really matter (to me)
Would just be amazing to be able to customize them outside the UI, this way I could define them in a seed script and even building a custom UI which could even handle i18n like mentioned in https://github.com/supabase/gotrue/issues/80#issuecomment-1239403830

Are they stored in Postgres? If so than it should be possible to edit them via SQL right, even before a "dedicated" API?

**Comment by GaryAustin1** [comment]
If you search templates here in discussions you will find many related discussions.
This is also an interesting thread on the subject... https://github.com/supabase/gotrue/issues/304

**Reply by CanRau** [reply]
for reference (for others) I posted this here because of this [discord comment](https://discord.com/channels/839993398554656828/1024854535874818113/1025042344883519520)

Thanks a lot tho for linking me to that gotrue issue and yea [generateLink](https://supabase.com/docs/reference/javascript/auth-api-generatelink) is what (I think) I'm going to use now, though being able to manipulate templates programmatically would still be interesting in general.

If the UI potentially provides [i18n support](https://github.com/supabase/gotrue/issues/80) for templates one could manage them via the UI (or API) then use `generateLink` to handle things "manually" while still using those email templates when sending.

Think this would allow for great customization for advanced work flows 🔥

**Reply by logemann** [reply]
How did you solve, that even when using your own email template engine, Supabase still sends emails the old fashioned way. There is imo no setting which will disable the supabase internal email sending no?
